pkg/api/graph/interfaces.go

```diff
@@ -51,6 +55,58 @@ func (m Markers) BySeverity(severity Severity) []Marker {
 	return ret
 }
 
+// FilterMissingNodes returns all the markers that have missing nodes from other namespace
+// and all those namespaces.
+func (m Markers) FilterMissingNodes(currentNamespace string) (Markers, []string) {
+	// Create a subgraph with all involved nodes.
+	g := New()
+	for i := range m {
+		g.AddNode(m[i].Node)
+		for r := range m[i].RelatedNodes {
+			g.AddNode(m[i].RelatedNodes[r])
```

Comment:
Unfortunately this panics on node id collisions :( I think I will have to filter out nodes with the same ID by myself...